eventmesh-webhook/eventmesh-webhook-admin/src/main/java/org/apache/eventmesh/webhook/admin/FileWebHookConfigOperation.java

```diff
@@ -64,7 +64,7 @@ public Integer insertWebHookConfig(final WebHookConfig webHookConfig) {
         if (!webHookConfig.getCallbackPath().startsWith(WebHookOperationConstant.CALLBACK_PATH_PREFIX)) {
             if (log.isErrorEnabled()) {
                 log.error("webhookConfig callback path must start with {}",
-                    WebHookOperationConstant.CALLBACK_PATH_PREFIX);
+                        WebHookOperationConstant.CALLBACK_PATH_PREFIX);
             }
```

Comment:
L66 and L67 should be merged into one line. I think the spotless plugin does not suppurt this feature.

Reply:
Some places like this has been merged into one line, such as `WebHookMQProducer`:

![image](https://github.com/apache/eventmesh/assets/41445332/e9f77617-a100-4258-9d14-6e15cc9143fd)

Reply:
> Some places like this has been merged into one line, such as `WebHookMQProducer`:
> 
> ![image](https://user-images.githubusercontent.com/41445332/268835228-e9f77617-a100-4258-9d14-6e15cc9143fd.png)

@Pil0tXia Pi The bug of indentation has been fixed in the latest commit.